app/App.html

```diff
@@ -0,0 +1,31 @@
+<Nav page={
```

Comment:
@Rich-Harris why the intensive `props.path` logic here? This seems very **manifest**-y, which goes against the grain of the automated path-finding you seem to support? What exactly is the function of these page props?

Reply:
In other words, do I have to add to this for every page I add?

Reply:
you can pass `props.path` down to `<Nav>` instead (in fact, I prefer that — will change it)...

```html
<Nav path={props.path}/>
```

...but yes, you have to explicitly pass stuff down, just like you used to have to explicitly pass down a `page` property via the `<Layout>` component in the previous template. How else will the nav bar know which option to mark as active?